### PR TITLE
Db 427 fix templating

### PR DESCRIPTION
**Pull request by jworcestBAH**
@bsweger, @nmonga91: This PR improves handling of missing values in templates, and fixes the typo that caused the error with titles in the admin registration notification

### CODE REVIEW

File: dataactbroker/handlers/aws/sesEmail.py [hunk 1]
```diff
@@ -27,9 +27,9 @@ def __init__(self,toAddress,fromAddress,content="",subject="",templateType=None,
             self.content = template.content
 
             for key in parameters :
-                if(not parameters[key] is None) :
+                if key in parameters and parameters[key] is not None:
```
Comment (bsweger) [inline dataactbroker/handlers/aws/sesEmail.py]: Is the key in parameters check needed here?

**Comment by bsweger** [comment]
@jworcestBAH just one quick question...

File: dataactbroker/handlers/aws/sesEmail.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactbroker/handlers/aws/sesEmail.py]: Removed the redundant check.

**Comment by bsweger** [comment]
Looks good!

**Pull request merged**